config/charge_controller.js

```diff
@@ -13,6 +13,5 @@ module.exports = {
     success: 'AUTHORISED',
     declined: 'DECLINED',
     error: 'ERROR'
-  },
-  preserveProperties: ['cardholderName', 'addressLine1', 'addressLine2', 'addressCity', 'addressPostcode', 'addressCountry', 'email']
```

Comment:
It might be a good idea to keep the `preserveProperties` method of picking fields. This would just stop future sensitive fields from being merged implicitly.

Reply:
Good point